/**:
  ros__parameters:
    # Target Detection Parameters:
    default_color: red
    base_frame: base
    gripper_frame: fer_hand_tcp
    color_frame: calibrated_colors.yaml
    # Calibration parameters in meters (m)
    calibration:
      tags:
        tag_1:
          x: -0.1106
          y: 0.446024
        tag_2:
          x: 0.7657
          y: 0.446024
        tag_3:
          x: 0.7657
          y: -0.514604
        tag_4:
          x: -0.1106
          y: -0.514604